Library/Homebrew/dev-cmd/pull.rb

```diff
@@ -321,6 +355,18 @@ def pull_patch
       apply_patch
     end
 
+    def pull_merge_commit(issue)
+      # Used by forks of homebrew-core that use merge-commits (for example linuxbrew)
+      ohai "Fast-forwarding to the merge commit"
+      test_bot_origin = patch_url[%r{(https://github\.com/[\w-]+/[\w-]+)/compare/}, 1]
+      safe_system "git", "fetch", "--quiet", test_bot_origin, "pr-#{issue}" if test_bot_origin
+      system "git", "merge", "--quiet", "--ff-only", "--no-edit", "FETCH_HEAD"
+      unless $CHILD_STATUS.success?
+        safe_system "git", "reset", "--hard", "FETCH_HEAD"
+        odie "Not possible to fast-forward"
```

Comment:
Would be good to improve this message to include why/what a workaround is.

Reply:
I'll add an explanation of the most common issue which triggers this.